internal/extsvc/github/v3.go

```diff
@@ -436,11 +436,11 @@ func (c *V3Client) ListRepositoryCollaborators(ctx context.Context, owner, repo
 	if len(affiliation) > 0 {
 		path = fmt.Sprintf("%s&affiliation=%s", path, affiliation)
 	}
-	_, err := c.get(ctx, path, &users)
+	responseState, err := c.get(ctx, path, &users)
```

Comment:
Nitpick here and in the other places: I think `respState` is long enough to make sense, but also nicely short.

```suggestion
	respState, err := c.get(ctx, path, &users)
```